docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/miro-source.md

```diff
@@ -0,0 +1,69 @@
+---
+id: miro-source
+title: Miro Source
+sidebar_label: Miro Source
+description: Configure Miro Source Cloud-to-Cloud connector
+---
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+
+The Miro Source API integration ingests audit logs obtained from the [Audit log API](https://developers.miro.com/reference/get-logs). You will be able to collect audit logs activity from your Miro platform using our new Miro Cloud-to-Cloud Source connector.
+
+## Data Sources
+
+The Miro API integration consumes data of the Audit Logs and sends it to Sumo Logic. The polling interval time for integration is 5 minutes.
+
+## Metadata
+
+If the Source is configured with the **SIEM forward** option, the metadata field `_siemparser` will be set.
+
+| Field Name | Value |
+| --- | --- |
+| _siemparser | #TODO |
+
+## Prerequisite
+
+To get audit logs, you must have Miro Enterprise account.
```

Comment:
To get audit logs, you must have a Miro Enterprise account.

Reply:
that a is missing.

Reply:
@stacykor Yeah that's a miss. Thank you for pointing that out.